src/duration/__tests__/format.ts

```diff
@@ -0,0 +1,180 @@
+    expect(duration({years: 0, seconds: -45}).toISOString()).toBe('PT-45S');
+    expect(duration({years: -5, seconds: 34}).toISOString()).toBe('P-5YT34S');
+});
+
+test('Duration#toISOString handles zero durations', () => {
+    expect(duration(0).toISOString()).toBe('PT0S');
+});
+
+// test('Duration#toISOString returns null for invalid durations', () => {
+//     expect(Duration.invalid('because').toISOString()).toBe(null);
+// });
+
+test('Duration#toISOString handles milliseconds duration', () => {
+    expect(duration({milliseconds: 7}).toISOString()).toBe('PT0.007S');
+});
+
+test('Duration#toISOString handles seconds/milliseconds duration', () => {
+    expect(duration({seconds: 17, milliseconds: 548}).toISOString()).toBe('PT17.548S');
+});
+
+test('Duration#toISOString handles negative seconds/milliseconds duration', () => {
+    expect(duration({seconds: -17, milliseconds: -548}).toISOString()).toBe('PT-17.548S');
+});
+
+test('Duration#toISOString handles mixed negative/positive numbers in seconds/milliseconds durations', () => {
+    expect(duration({seconds: 17, milliseconds: -548}).toISOString()).toBe('PT16.452S');
+    expect(duration({seconds: -17, milliseconds: 548}).toISOString()).toBe('PT-16.452S');
+});
+
+//------
+// #toJSON()
+//------
+
+test('Duration#toJSON returns the ISO representation', () => {
+    expect(dur().toJSON()).toBe(dur().toISOString());
+});
+
+//------
+// #toString()
+//------
+
+test('Duration#toString returns the ISO representation', () => {
+    expect(dur().toString()).toBe(dur().toISOString());
+});
+//------
+// #humanizeIntl()
+//------
+
+test('Duration#humanizeIntl formats out a list', () => {
+    expect(dur().humanizeIntl()).toEqual(
+        '1 year, 2 months, 1 week, 3 days, 4 hours, 5 minutes, 6 seconds, 7 milliseconds',
+    );
+});
+
+test('Duration#humanizeIntl only shows the units you have', () => {
+    expect(duration({years: 3, hours: 4}).humanizeIntl()).toEqual('3 years, 4 hours');
+});
+
+test('Duration#humanizeIntl accepts a listStyle', () => {
+    expect(dur().humanizeIntl({listStyle: 'long'})).toEqual(
+        '1 year, 2 months, 1 week, 3 days, 4 hours, 5 minutes, 6 seconds, and 7 milliseconds',
+    );
+});
+
+test('Duration#humanizeIntl accepts number format opts', () => {
+    expect(dur().humanizeIntl({unitDisplay: 'short'})).toEqual(
+        '1 yr, 2 mths, 1 wk, 3 days, 4 hr, 5 min, 6 sec, 7 ms',
+    );
+});
+
+test('Duration#humanizeIntl works in differt languages', () => {
+    expect(dur().locale('fr').humanizeIntl()).toEqual(
+        '1 an, 2 mois, 1 semaine, 3 jours, 4 heures, 5 minutes, 6 secondes, 7 millisecondes',
+    );
+});
+
+//------
+// #humanize()
+//------
+
+test('Duration#humanize', () => {
+    expect(duration({seconds: 44}).humanize()).toBe('a few seconds');
+    expect(duration({seconds: 45}).humanize()).toBe('a minute');
+    expect(duration({seconds: 89}).humanize()).toBe('a minute');
```

Comment:
Is there an option to display it like `1 minute` instead of `a minute`?

Reply:
No. For fixed numbers you can use `humanizeIntl`, but it gives exact numbers e.g. `45 seconds`